roles/cri-o/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Add OpenShift Origin repository
+  yum_repository:
+    name: origin
+    description: OpenShift Origin Repo
+    baseurl: "{{ crio_rhel_repo_base_url }}"
+    gpgcheck: no
+  when:
+    - ansible_os_family == 'RedHat'
+
+- name: Install cri-o
+  package:
```

Comment:
is it working on debian/suse/ubuntu/coreos?

Reply:
I want to just support for centos or redhat related distribution. If I won't port to other distribution, should I cancel PR?

Reply:
no but the pattern should take that fact. The docker role is very os specific, but mostly every task are generic and the split is made via vars/{{distribution}}
this way following contributions will have a clear way to add other OS

Reply:
I made separate vars and define redhat specific parameters as docker roles.
repository url is keeped in defaults/main.yml because docker roles has repository url in defaults/main.yml.